### PR TITLE
Update the pull-waveform script 

### PR DESCRIPTION
**Pull request by Benz-Poobua**
- Add pull-waveform script, fetching data from dataframe, evid, IRIS client
- Add demo notebook

### CODE REVIEW

File: src/notebooks/demo.ipynb
Comment (nts345045) [inline src/notebooks/demo.ipynb:1]: Rename to describe what it is a demo of and add a small markdown block at the top of the notebook describing what the notebook does and which *.py script it is using for methods

**Review by nts345045 — CHANGES_REQUESTED** [review]
Update demo.ipynb per comment

**Comment by Benz-Poobua** [comment]
- Add the pull_wf_by_evid.py and comments
- Add the tutorial file using pull_wf_by_evid module and necessary comments

**Review by nts345045 — APPROVED** [review]
Revisions complete

**Pull request merged**